gui/wxpython/gui_core/infobar.py

```diff
@@ -0,0 +1,89 @@
+
+import wx
+import wx.aui
+try:
+    import wx.lib.agw.infobar as IB
+except ImportError:
+    import wx.lib.infobar as IB
+
+
+class InfoBar(IB.InfoBar):
+    """
+    Custom Info bar
+    """
+    def __init__(self, parent):
+        IB.InfoBar.__init__(self, parent)
+
+        colBg = wx.SystemSettings.GetColour(wx.SYS_COLOUR_HIGHLIGHT)
+        self.SetBackgroundColour(colBg)
+        self.SetOwnForegroundColour(wx.SystemSettings.GetColour(wx.SYS_COLOUR_HIGHLIGHTTEXT))
+        self._text.SetBackgroundColour(colBg)
+        self._button.SetBackgroundColour(colBg)
```

Comment:
Set both bg and fg colors. The text has to have highlight text color when the bg is highlight color.

```suggestion
self._background_color = wx.SystemSettings.GetColour(
            wx.SYS_COLOUR_HIGHLIGHT
        )
        self._foreground_color = wx.SystemSettings.GetColour(
            wx.SYS_COLOUR_HIGHLIGHTTEXT
        )
        self.SetBackgroundColour(self._background_color)
        self.SetForegroundColour(self._foreground_color)
        self._text.SetBackgroundColour(self._background_color)
        self._text.SetForegroundColour(self._foreground_color)
        self._button.SetBackgroundColour(self._background_color)
```

Doing `SetForegroundColour` for self, although I don't see any effect one way or the other. (The original code had `SetOwnForegroundColour`.)